helix-term/src/ui/menu.rs

```diff
@@ -300,14 +303,24 @@ impl<T: Item + 'static> Component for Menu<T> {
         Some(self.size)
     }
 
-    fn render(&mut self, area: Rect, surface: &mut Surface, cx: &mut Context) {
+    fn render(&mut self, mut area: Rect, surface: &mut Surface, cx: &mut Context) {
         let theme = &cx.editor.theme;
         let style = theme
             .try_get("ui.menu")
             .unwrap_or_else(|| theme.get("ui.text"));
         let selected = theme.get("ui.menu.selected");
         surface.clear_with(area, style);
 
+        let border_config = &cx.editor.config().popup_border;
+
+        let render_borders =
+            border_config == &PopupBorderConfig::All || border_config == &PopupBorderConfig::Menu;
+
+        if render_borders {
+            Widget::render(Block::default().borders(Borders::ALL), area, surface);
+            area = area.clip_top(1).clip_bottom(1);
```

Comment:
area.inner doesn't work with Borders?

Reply:
It does, i changed it to use area.inner now.

Reply:
I removed the mut from area in function parameter and instead create a new variable "inner" to make it look more consistent with popup render code.
If you prefer the previous version i can revert this change.